src/pocket/PocketPagerDuty.ts

```diff
@@ -133,7 +133,7 @@ export class PocketPagerDuty extends Resource {
   ): ServiceIntegration {
     return new ServiceIntegration(
       this,
-      `${vendor.name}-${urgency.toLowerCase()}`,
+      `${vendor.friendlyUniqueId}-${urgency.toLowerCase()}`,
```

Comment:
Note Terraform fixed a bug. 

https://stackoverflow.com/questions/61957767/aws-cdk-cannot-use-tokens-in-construct-id-how-do-i-dynamically-name-construc

https://bobbyhadz.com/blog/aws-cdk-cannot-use-tokens-construct-id

Reply:
This will cause a few NON-Critical resources to want to recreate when we merge into other repos. It should be limited to pagerduty integrations and autoscaling policies.

Reply:
can/should we test this in dev to ensure there aren't any unforeseen consequences? specifically thinking about the change to the dynamo index above.

Reply:
yea agreed. I think you caught the big one, not concerned about pagerduty re-creating.